extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -287,6 +287,7 @@ export class SubscriptionService {
         'Publication id is null'
     ));
     const origin = getWinOrigin(this.ampdoc_.win);
+    const cdnOrigin = getSourceOrigin(this.ampdoc_.win.location);
```

Comment:
The code correct, but this origin is actually never "CDN". It's always direct 3p origin. The window origin could either be a 3p or CDN, but not this one.